Cargo.lock

```diff
@@ -981,9 +930,9 @@ dependencies = [
 
 [[package]]
 name = "crossbeam-utils"
-version = "0.8.5"
+version = "0.8.8"
```

Comment:
Please revert to 0.8.5

Reply:
This is required by crossbeam-channel 0.5.4, required by solana-metrics 1.11.0